src/benchmarks/micro/corefx/System.Text.Json/Utf8JsonWriter/Perf.HelloWorld.cs

```diff
@@ -0,0 +1,76 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Buffers;
+using BenchmarkDotNet.Attributes;
+using MicroBenchmarks;
+
+namespace System.Text.Json
+{
+    [BenchmarkCategory(Categories.CoreFX, Categories.JSON)]
+    public class Perf_HelloWorld
```

Comment:
Should this class be `Perf_String` instead?

Reply:
We already have `Perf_Strings` below. I would remove this entirely.

Reply:
Agreed. I just got to it.

Reply:
The "hello world" scenario is relevant for tech empower (which is why I tend to keep such a test throughout the JSON stack) and I wanted to highlight that/be able to run it separately.

If I am not hitting the benchmark limit, I am OK with moving it in.

Reply:
Techempower is covered by the asp.net lab. In this repo we'd like the smallest set of benchmarks that cover a feature as we can have. We don't have a specific limit for benchmarks. We encourage avoiding obvious duplication, which this seems to be. If there's a case where this test will show a regression that the tests in `Perf_Strings` does not show, that sounds like a missing case in `Perf_Strings` to me. (In this case I do not believe that would be likely.)

Reply:
Makes sense to me. Will remove for now.